<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="jemok">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>641</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:96pt; font-weight:600;&quot;&gt;Trip Planner&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelindi">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;출발지를 입력해주세요&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>입력</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imnyukchang">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="undo_Btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toKyoungyu">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>경유지 입력하러가기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>입력</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelindi_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;차량정보를 입력해주세요&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrip_Planner">
    <property name="title">
     <string>Trip Planner</string>
    </property>
    <widget class="QMenu" name="menuQuit">
     <property name="title">
      <string>Quit</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuQuit"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuTrip_Planner"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
